changelogs/8.11.asciidoc

```diff
@@ -0,0 +1,46 @@
+[[release-notes-8.11]]
+== APM version 8.11
+
+https://github.com/elastic/apm-server/compare/8.10\...8.11[View commits]
+
+* <<release-notes-8.11.0>>
+
+[float]
+[[release-notes-8.11.0]]
+=== APM version 8.11.0
+
+[float]
+==== Breaking Changes
+- The `ecs.version` field has been removed from APM data streams {pull}11632[11632]
+
+[float]
+==== Deprecations
+
+[float]
+==== Bug fixes
+- Add back gzip support for grpc otlp endpoint {pull}11434[11434]
+- Correctly mark jvm.memory.non_heap.pool.* and jvm.fd.* metrics as internal {pull}11303[11303]
+- Fix tail-based sampling discarding low throughput and low sample rate traces {pull}11642[11642]
+- Add memory based autoscaling for service destination aggregation groups {pull}11739[11739]
+
+[float]
+==== Intake API Changes
+
+[float]
+==== Added
+- Support and define DLM data retention period in the apmpackage {pull}11539[11539]
+- Expose new metrics into the local batch processor {pull}11582[11582] :
+	- http.server.request.count
+	- http.server.request.duration
+	- http.server.response.valid.count
+	- http.server.response.errors.count
+	- http.server.errors.timeout
+	- http.server.errors.ratelimit
+	- grpc.server.request.count
+	- grpc.server.request.duration
+	- grpc.server.response.valid.count
+	- grpc.server.response.errors.count
+	- grpc.server.errors.timeout
+	- grpc.server.errors.ratelimit
```

Comment:
I would not list the metrics here, they can be found in the PR

Reply:
+1, in that case we'll have to change the changelog in 8.11 branch as well

Reply:
https://github.com/elastic/apm-server/pull/11767